github/repos.go

```diff
@@ -529,6 +531,12 @@ type RequiredStatusChecks struct {
 	Contexts *[]string `json:"contexts,omitempty"`
 }
 
+// RequiredPullRequestReviews represents the protection configuration for pull requests.
+type RequiredPullRequestReviews struct {
+	// Enforce pull request reviews for repository administrators.
+	IncludeAdmins *bool `json:"include_admins,omitempty"`
```

Comment:
`include_admins` key is [documented](https://developer.github.com/v3/repos/branches/#update-branch-protection) as:

> **Required.** Enforce required status checks for repository administrators.

Shouldn't we not use `omitempty` option for required parameters?